### PR TITLE
[ra2] ch5 re-word

### LINKED ISSUE #1581: [RA2 Ch5] Change wording from requirement style to guidance style
As per agreement in the vF2F, this chapter is for security guidance only (i.e. wording should be e.g. "it is recommended that...") and not specifications, which will need to go in chapter04.md

### PR DESCRIPTION
**Pull request by tomkivlin**
Remove "must" as this chapter doesn't form part of the specification, it is guidance only.

Note this is grammar/typos only - not asking for a review of wording (please open new issues for that).

Resolves #1581

### CODE REVIEW

**Review by petorre — APPROVED** [review]

File: doc/ref_arch/kubernetes/chapters/chapter05.md [hunk 1]
```diff
@@ -4,176 +4,275 @@
...
-<a name="5.2"></a>
-##  5.2 Principles
+## 5.2 Principles
 The following are core principles to consider when securing cloud native applications:
 
 - Deploy only secure applications and trusted codes
 - Only deploy applications from validated and verified images
 - Only deploy applications from trusted registries
-- Containers orchestration (Kubernetes) must be secure with administrative boundaries between tenants
+- Containers orchestration (Kubernetes) must be secure with administrative boundaries
```
Comment (xmulligan) [inline doc/ref_arch/kubernetes/chapters/chapter05.md:57]:
```suggestion
- Containers orchestration (Kubernetes) secured with administrative boundaries
```

File: doc/ref_arch/kubernetes/chapters/chapter05.md [hunk 2]
```diff
@@ -4,176 +4,275 @@
...
+## 5.12  Secrets Management
+It is recommended that the principle of least privilege is applied to secret
+management in Kubernetes:
+- Ensure that the cloud native applications can only read the secrets that these
+applications need
+- Have different set of secrets for different environments(like production,
+development, and testing)
+
+Secret values protect sensitive data, it is recommended to protect them from
+unauthorised access. Ideally, they must be protected at rest and in transit.
```
Comment (xmulligan) [inline doc/ref_arch/kubernetes/chapters/chapter05.md:142]:
```suggestion
Secret values protect sensitive data, it is recommended to protect them from
unauthorised access. Ideally, by being protected at rest and in transit.
```

File: doc/ref_arch/kubernetes/chapters/chapter05.md [hunk 3]
```diff
@@ -4,176 +4,275 @@
...
 
-- Cluster management Network isolation can help protect the master node and control where administrative commands can run. Use network isolation techniques, configure RBAC on the cluster manager and configure node service accounts following the principle of least privilege.
-- Ensure that access control is applied to registries requiring unique credentials, to limit who can control the build or add images.
+- Cluster management Network isolation can help protect the master node and
+control where administrative commands can run. Use network isolation techniques,
+configure RBAC on the cluster manager and configure node service accounts
+following the principle of least privilege.
+- Ensure that access control is applied to registries requiring unique credentials,
+to limit who can control the build or add images.
 - Network access must run over TLS connections.
```
Comment (xmulligan) [inline doc/ref_arch/kubernetes/chapters/chapter05.md:201]:
```suggestion
- Network access runs over TLS connections.
```

File: doc/ref_arch/kubernetes/chapters/chapter05.md [hunk 4]
```diff
@@ -4,176 +4,275 @@
...
+- **Containers**: A lightweight and portable executable image that contains
+software and all of its dependencies. The containers need to be prevented from
+accessing the underlying OS like loading of kernel modules, mounting of
+directories of underlying OS etc and it must also be ensured that they don't
+run in privileged mode.
+- **Pods**: A Pod represents a set of running containers on your cluster.
+Kubernetes inherently offers pod security policies that define a set of
+conditions that a pod must run with in order to be accepted into the system.
+These policies help in ensuring the necessary checks for running the pods.
+- **Kubernetes Node**: A Kubernetes node in an unsecured boundary can lead to a
```
Comment (xmulligan) [inline doc/ref_arch/kubernetes/chapters/chapter05.md:244]:
```suggestion
software and all of its dependencies. The containers need to be prevented from
accessing the underlying OS like loading of kernel modules, mounting of
directories of underlying OS etc and ensuring that they don't
run in privileged mode.
- **Pods**: A Pod represents a set of running containers on your cluster.
Kubernetes inherently offers pod security policies that define a set of
conditions that a pod needs to run with in order to be accepted into the system.
These policies help in ensuring the necessary checks for running the pods.
- **Kubernetes Node**: A Kubernetes node in an unsecured boundary can lead to a
```

**Review by xmulligan — COMMENTED** [review]
removed a few more musts

**Comment by tomkivlin** [comment]
Thanks, @xmulligan - all suggestions committed.

**Review by ulikleber — APPROVED** [review]

**Review by xmulligan — APPROVED** [review]

**Comment by tomkivlin** [comment]
Merging as have 3 approvals.

**Pull request merged**